website/pages/menu.mdx

```diff
@@ -70,7 +70,7 @@ the Menu component. `isOpen` reflects the current open or closed state of the `C
 ```vue live=true
 <template>
   <c-menu v-slot="{ isOpen }">
-    <c-menu-button :is-active="isOpen" right-icon="chevron-down">
+    <c-menu-button :is-active="isOpen" :right-icon="isOpen ? 'chevron-up' : 'chevron-down'" right-icon="chevron-down">
```

Comment:
Also since we have the conditional prop this can be simplified to just one `:right-icon` prop
```suggestion
    <c-menu-button :is-active="isOpen" :right-icon="isOpen ? 'chevron-up' : 'chevron-down'">
```